spec/models/verse_spec.rb

```diff
@@ -58,4 +58,13 @@
       )
     end
   end
+
+  describe '.verse_of_the_day' do
+    it 'returns the correct verse' do
+      date = (Date.today.yday % 100) + 1
+      verse = create(:verse, day: date)
+
+      expect(Verse.verse_of_the_day.scripture).to eq(verse.scripture)
```

Comment:
You could drop the `scripture` part here, you already have the verse as a variable here so you can just `expect(Verse.verse_of_the_day).to eq(verse)`

Reply:
Ahh okay makes sense!! 🙌